src/ReduxStore/FeedDetails/feedReducers.js

```diff
@@ -41,12 +47,20 @@ export const feedReducer = (state = initialState, action) => {
     case TAG_RELATED_ARTICLE_LOADED:
       return { ...state, tagRelatedArticles: action.tagRelatedArticles };
 
-    case POPULAR_TAG_DISPLAYED:
-      return { ...state, isDisplay: action.tagName };
-
     case RELATED_TAG_LOADED:
       return { ...state, currentTagName: action.tagName };
 
+    case GLOBLE_FEED_CLICKED:
+      return { ...state, currentTagName: "", tagRelatedArticles: [] };
```

Comment:
Set `tagRelatedArticles` to empty array will cause problem when clicking global feed link

Reply:
fixed